server/lib/src/runner.dart

```diff
@@ -9,6 +9,7 @@ import 'package:integration_test/integration_test.dart';
 import 'package:package_info_plus/package_info_plus.dart';
 
 const MAX_TEST_DURATION_SECS = 24 * 60 * 60;
+const SERVER_VERSION = '0.0.14';
```

Comment:
I believe the version number can be retrieved dynamically: https://stackoverflow.com/questions/53672171/how-to-get-build-and-version-number-of-flutter-app